docs/src/Module.doc.js

```diff
@@ -103,21 +104,36 @@ card(
         ],
       },
       {
-        name: 'onExpandedChange',
-        type: '(?number) => void',
-        required: false,
-        description: [
-          'This callback is executed whenever any module item is expanded or collapsed. It receives the index of the currently expanded module, or null if none are expanded.',
-        ],
+        name: 'id',
+        href: 'expandable-default',
+        type: 'string',
+        required: true,
+        description: 'Unique id to identify this Module',
       },
       {
         name: 'items',
         href: 'expandable-items',
-        type:
-          'Array<{| title: string, icon?: $Keys<typeof icons>, summary?: Array<string>, type?: "info" | "error", iconAccessibilityLabel?: string, children: ?React.Node |}>',
+        type: `
+        Array<{|
```

Comment:
Sadly this doesn't affect how the type is rendered, but it does make it a lot more readable here